db/data/development-data/index.js

```diff
@@ -2,3 +2,5 @@ exports.articleData = require('./articles.js');
 exports.commentData = require('./comments.js');
 exports.topicData = require('./topics.js');
 exports.userData = require('./users.js');
+
+module.exports = {articleData, commentData, topicData, userData}
```

Comment:
You don't need this line! The exports have already been taken care of.

Reply:
Ok thanks . Do i need to take the line out and re push it to the setup branch?